tests/git/test_git_commands.py

```diff
@@ -2,24 +2,32 @@
 from textwrap import dedent
 from unittest import mock
 
-import pytest
 from snowflake.connector import ProgrammingError
 
 EXAMPLE_URL = "https://github.com/an-example-repo.git"
 
 
-@pytest.mark.skip(reason="Command is hidden")
 def test_toplevel_help(runner):
     result = runner.invoke(["--help"])
     assert (
         result.exit_code == 0
         and "Manages git repositories in Snowflake." in result.output
     )
-
     result = runner.invoke(["git", "--help"])
     assert result.exit_code == 0, result.output
 
 
+def test_not_visible_if_disabled(runner, monkeypatch):
+    monkeypatch.setenv("SNOWFLAKE_CLI_FEATURES_ENABLE_SNOWGIT", False)
```

Comment:
Let's stick to already used approach
https://github.com/snowflakedb/snowflake-cli/blob/2b8558c236fff1809148df1d0d283ecbcb42aa33/tests/api/test_feature_flags.py#L54-L56

Reply:
In this case I would recommed using it as decorator